### PR TITLE
new magic effect classes constants

### PR DESCRIPTION
**Pull request by raymondtfr**

### CODE REVIEW

File: src/const.h [hunk 1]
```diff
@@ -108,6 +108,11 @@ enum MagicEffectClasses : uint8_t {
 	CONST_ME_YELLOWSMOKE = 168,
 	CONST_ME_GREENSMOKE = 169,
 	CONST_ME_PURPLESMOKE = 170,
+	CONST_ME_EARLY_THUNDER = 171,
+	CONST_ME_RAGIAZ_BONECAPSULE = 172,
+	CONST_ME_CRITICAL_DAMAGE = 173,
+
```
Comment (ranisalt) [inline src/const.h]: What about 174?

Comment (raymondtfr) [inline src/const.h]: It's empty.

Comment (ranisalt) [inline src/const.h]: So mark it just like above (see 159-166)

**Comment by ranisalt** [comment]
Thanks!

**Pull request merged**